taui/src/components/edit-profile.js

```diff
@@ -268,87 +269,86 @@ export default class EditProfile extends PureComponent<Props> {
-            className='account-profile__button account-profile__button--secondary'
-            data-id={index}
-            onClick={(e) => deleteAddress(index, e)}
-            title={message('Profile.DeleteAddress')}>
-            <img src='assets/trash-alt.svg' width='16' alt={message('Profile.Delete')} />
-          </button>
-        </div>
+        {showAllColumns && <>
+          <div className='account-profile__destination_field account-profile__destination_field--narrow account-profile__destination_field--center'>
+            <input
+              className='account-profile__input account-profile__input--radio'
+              id='primary'
+              type='radio'
+              onChange={(e) => setPrimaryAddress(index, e)}
+              checked={!!destination.primary}
+            />
+          </div>
+          <div className='account-profile__destination_field account-profile__destination_field--xnarrow account-profile__destination_field--center'>
+            <button
+              id='deleteAddress'
+              className='account-profile__destination-delete-button'
+              data-id={index}
+              onClick={(e) => deleteAddress(index, e)}
+              title={message('Profile.DeleteAddress')}>
+              <Icon type='times' />
+            </button>
+          </div>
+        </>}
       </li>
     })
 
     return (
-      <div className=''>
-        <ul className=''>
-          <li
-            key='header'
-            className='account-profile__destination_row'>
-            <div className='account-profile__destination_field'>
-              <label
-                className='account-profile__label'
-                htmlFor='address'>
-                {message('Profile.Address')}
-              </label>
-            </div>
-            <div className='account-profile__destination_narrow_field'>
-              <label
-                className='account-profile__label'
-                htmlFor='purpose'>
-                {message('Profile.Purpose')}
-              </label>
-            </div>
-            <div className='account-profile__destination_narrow_field'>
-              <label
-                className='account-profile__label'
-                htmlFor='primary'>
+      <div className='account-profile__destinations'>
+        <h2 className='account-profile__label'>{message('Profile.Destinations')}</h2>
+        <div className='account-profile__destination-list-header'>
+          <div className='account-profile__destination_field account-profile__destination_field--wide'>
+            <span className='account-profile__destination-list-heading'>
+              {message('Profile.Address')}
+            </span>
+          </div>
+          <div className='account-profile__destination_field'>
+            <span className='account-profile__destination-list-heading'>
+              {message('Profile.Purpose')}
+            </span>
+          </div>
+          {showAllColumns && <>
```

Comment:
Interesting, hadn't run into [shortened fragment declaration](https://reactjs.org/docs/fragments.html#short-syntax) before.

Reply:
Right? It's great.